weblogic-azure-aks/pom.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+
+    Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
The project is license using UPL as you used in other files in this PR. Why the Apache license here?

Reply:
ah...I copied the file from original wls-vm offer, and forgot to fix it, thanks for you finding :)

Reply:
Fixed in https://github.com/galiacheng/weblogic-azure/commit/6eeb6ca7cc839d72873352d773f6a343bb294a38